README.md

```diff
@@ -136,28 +134,17 @@ cd server
 
 cp src/.mf.config.SAMPLE.json src/.mf.config.json
 
+# To install dependencies, clean/create build/ subdirectory, build and upgrade SDK, build source code within this project
 npm install
-npm run clean             # Cleans/creates build/ subdirectory
```

Comment:
This is still required for discovery.

Reply:
```
   "upgradeSDKAndBuildMF": "npm update @firebolt-js/discovery-sdk && npm run clean && npm run build:discovery && npm run build:mf",
    "postinstall": "npm run upgradeSDKAndBuildMF"
```
I have only removed the core and manage steps, all the remaining steps(related to discovery,clean and create build folder, building src code) are moved into a postinstall script in server/package.json as above. Thought this would be better and  easy on the user as user now only has to run 'npm install' manually instead of each of the steps and rest will be taken care by the same command in the background